playbooks/operationalizing/ingress.adoc

```diff
@@ -0,0 +1,202 @@
+---
+---
+= Using Ingress to Expose Services
+Andrew Block <ablock@redhat.com>
+v1.0, 2016-11-07
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+Using OpenShift's ingress capabilities to provide access to services running on the cluster
+
+
+toc::[]
+
+
+== Abstract
+
+OpenShift provides multiple methods for communicating with services running on the platform. The most common method is to use the integrated haproxy router solution. While this option provides support for the majority of use cases, there are restrictions which limit the usage by certain applications. Fortunately, OpenShift provides several alternate options, such as ingress, for exposing and accessing services without the use of the haproxy router. 
+
```

Comment:
Considering the complexity here, I think this doc calls for some kind of diagram. If you'd like we can merge without and it can be a followup issue.